src/main/java/com/searchservice/app/rest/errors/CustomExceptionHandler.java

```diff
@@ -12,11 +14,14 @@
 @AllArgsConstructor
 @EqualsAndHashCode(callSuper = false)
 @Component
-public class TableNotUnderDeletionException extends RuntimeException{
-
+public class CustomExceptionHandler extends RuntimeException{
```

Comment:
Would be better if we name it as "Exception" only.. like, CustomException/CustomSAASException